RPIPs/RPIP-43.md

```diff
@@ -77,24 +83,21 @@ into shares is defined in [RPIP-46](RPIP-46.md).
 - Rewards distribution MAY be temporarily blocked while validators are exiting or
   pending removal.
 - There SHALL be a reward distribution function in the megapool
-  - When called, rewards from the node operator's bonded ETH SHALL be held in
-    the megapool as unclaimed node operator funds.
-  - When called, `node_operator_commission_share` of rewards SHALL be held in
-      the megapool as unclaimed node operator funds.
   - When called, `reth_share` of rewards SHALL be sent to the rETH contract
   - When called, `voter_share` of rewards SHALL be sent to a merkle rewards
     distributor contract
   - When called, `surplus_share` of rewards SHALL be sent to the appropriate
     surplus disposition contract
```

Comment:
This reorder materially changed things and we need another little tweak. In particular, it's `reth_share` of the rewards from the borrowed ETH. The rewards from the bonded eth should not go to these shares.

Taking a stab at it:
```
- There SHALL be a reward distribution function in the megapool
  - For this section, we define `borrowed_share` as the megapool's `borrowed_eth / (bonded_eth + borrowed_eth)`
  - When called, `reth_share * borrowed_share` of rewards SHALL be sent to the rETH contract
  - When called, `voter_share * borrowed_share` of rewards SHALL be sent to a merkle rewards
    distributor contract
  - When called, `surplus_share * borrowed_share` of rewards SHALL be sent to the appropriate
    surplus disposition contract
  - When called while the megapool has `debt`, the remaining rewards SHALL first be used to pay off `debt`
   - When called, any remaining rewards SHALL then be held in the megapool as unclaimed node operator funds 
```

Reply:
not sure I like "borrowed_share" as a name since share is being used for something else here. borrowed_proportion?

Reply:
Note that I unindented the final bullet in that blurb one level. I think this works with "SHALL first"? The same pattern is used on lines 109/110 with similar indentation to my proposed blurb.